### PR TITLE
Igor Kovalenko: Create an instance of the Restaurant class and add a …

### PR DESCRIPTION
**Pull request by Igrekonko**
…few orders with multiple items each

### CODE REVIEW

**Review by RomanO-qa — APPROVED** [review]

File: restaurant_ordering_system_IKovalenko/appetizer.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+
```
Comment (toleksyn) [inline restaurant_ordering_system_IKovalenko/appetizer.py:2]: no extra lines

File: restaurant_ordering_system_IKovalenko/main_instance.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+from appetizer import Appetizer
+from entree import Entree
+from drink import Drink
+from order import Order
+from restaurant import Restaurant
+
+# Create some menu items
+appetizer1 = Appetizer("Mussels", 9.99, "Mussels In Lemon Garlic-Butter Sauce", True, True)
+entree1 = Entree("Borscht", 49.95, "Traditional Ukrainian Borscht", False, True)
+drink1 = Drink("Juice", 12.30, "Natural juices in the assortement", "Medium", True)
```
Comment (toleksyn) [inline restaurant_ordering_system_IKovalenko/main_instance.py:11]: rename, avoid 1,2 in variable names

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: restaurant_ordering_system_IKovalenko/main_instance.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+from order import Order
+from restaurant import Restaurant
+
+# Create some menu items
+mussels = Appetizer("Mussels", 9.99, "Mussels In Lemon Garlic-Butter Sauce", True, True)
+borscht = Entree("Borscht", 49.95, "Traditional Ukrainian Borscht", False, True)
+juice = Drink("Juice", 12.30, "Natural juices in the assortement", "Medium", True)
+
+# Create an order
+order1 = Order(1)
```
Comment (toleksyn) [inline restaurant_ordering_system_IKovalenko/main_instance.py:13]: first_order

File: restaurant_ordering_system_IKovalenko/main_instance.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+order2.add_item(drink2)
+
+# Create a restaurant
+my_restaurant = Restaurant("Family Food")
+my_restaurant.add_order(1, order1)
+my_restaurant.add_order(2, order2)
+
+# Print the orders in the restaurant
+orders = my_restaurant.get_orders()
+for table_number, order in orders.items():
```
Comment (toleksyn) [inline restaurant_ordering_system_IKovalenko/main_instance.py:34]: maybe makes sense to add print_orders(table_number) function to the Restaurant class?

**Review by toleksyn — CHANGES_REQUESTED** [review]

**Pull request closed without merging**